addons/website_slides/controllers/main.py

```diff
@@ -301,14 +304,17 @@ def _slide_channel_prepare_values(self, **kw):
         if kw.get('tag_ids'):
             tag_ids = [int(item) for item in kw['tag_ids'].split(',')]
 
-        return {
+        values = self._prepare_channel_values(**kw)
```

Comment:
not needed. Simply extend `_slide_channel_prepare_values`, it should be enough ;)